pkl-codegen-kotlin/src/test/kotlin/org/pkl/codegen/kotlin/KotlinCodeGeneratorTest.kt

```diff
@@ -553,6 +555,92 @@ class KotlinCodeGeneratorTest {
     assertCompilesSuccessfully(kotlinCode)
   }
 
+  @Test
+  fun `custom kotlin package prefix`() {
+    val kotlinCode =
+      generateKotlinCode(
+        """
+      module my.mod
+
+      class Person {
+        name: String
+        age: Int
+        hobbies: List<String>
+        friends: Map<String, Person>
+        sibling: Person?
+      }
```

Comment:
Very low-priority, but for other/future cases as well; we default to using bird examples. The Person instances you'll find were missed in cleanup.